### PR TITLE
dump indices before and after test

### PR DESCRIPTION
**Pull request by richm**
So we can find out of project.default is being created by some test
/test

### CODE REVIEW

**Comment by richm** [comment]
bingo - must be a test flake in the read-throttling test:
```
before /data/src/github.com/openshift/origin-aggregated-logging/hack/testing/test-remote-syslog.sh logging-es-data-master-d6ks5ubv-1-8vnxg indices
green open .searchguard.logging-es-data-master-d6ks5ubv                          1 0    5 0  40.9kb  40.9kb 
green open project.default.044be73a-b4fe-11e7-8927-0ec06334315e.2017.10.19       1 0    0 0    130b    130b 
```
although it doesn't show up after that test - it must take a few seconds for es to create the index

**Comment by richm** [comment]
I changed the ansible to explicitly pass in the value on the command line:
```
                         -e openshift_logging_fluentd_audit_container_engine=True                 \
                          /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/openshift-logging.yml \
```
We'll see if that makes a difference
/retest

File: hack/testing/entrypoint.sh [hunk 1]
```diff
@@ -113,6 +113,13 @@ function run_suite() {
 	os::cmd::expect_success "oc project logging"
 	os::test::junit::declare_suite_end
 
+    local es_pod=$( get_es_pod es )
```
Comment (ewolinetz) [inline hack/testing/entrypoint.sh]: im guessing once the uncommented out tests all pass we'll remove these sections?

Comment (richm) [reply hack/testing/entrypoint.sh]: They are passing now - you want me to remove the checking for the indices now?

Comment (ewolinetz) [reply hack/testing/entrypoint.sh]: Sure, other than that this seems good to go to me.

**Pull request merged**